man/docker-info.1.md

```diff
@@ -11,9 +11,9 @@ docker-info - Display system-wide information
 
 # DESCRIPTION
 This command displays system wide information regarding the Docker installation.
-Information displayed includes the number of containers and images, pool name,
-data file, metadata file, data space used, total data space, metadata space used
-, total metadata space, execution driver, and the kernel version.
+Information displayed includes the number of containers and (non-tagged) images,
```

Comment:
I'm guessing you really mean `all images (including those that are un-tagged)`

`(non-tagged) images` can be interpreted to mean that the count does not include tagged images.